<launch>
  <group ns="drive_motor">
    <rosparam file="$(find drive)/config/teensy_eday.yaml" />
  </group>
  <node pkg="drive" type="drive_4_motor.py" name="four_wheel_drive" output="screen"></node>
</launch>

